client/client-multi/client-c/src/kaa_common.h

```diff
@@ -66,26 +66,35 @@ typedef enum {
     KAA_SERVICE_LOGGING = 4,
 } kaa_service_t;
 
+/**
+ * @brief Unique identifier used to represent client transport channel implementations.
+ */
+typedef struct {
+    uint32_t id;
+    uint16_t version;
+} kaa_transport_protocol_info_t;
```

Comment:
Is this structure represents the protocol ID or the unique channel ID? If it is an unique ID of the channel (not a transport protocol like KaaTCP) then my comments above are not fully correct. And if so then you should rename it to smth like "kaa_channel_info_t", because current name doesn't match to description. 
NOTE: Kaa SDK can hold several channel implementations with the same transport protocols.